interface/therapy_groups/therapy_groups_views/listGroups.php

```diff
@@ -264,6 +265,19 @@
         });
     });
 
+    function oeFormatShortDate(value){
```

Comment:
Actually i just realized you are instead mimicking the function DateToYYYYMMDD, so would instead name this DateToYYYYMMDD_js